src/main/modules/properties-volume/index.ts

```diff
@@ -31,6 +31,7 @@ export class PropertiesVolume {
       this.setLocalSecret('AppInsightsConnectionString', 'appInsights.connectionString');
       this.setLocalSecret('launchdarkly-sdk-key', 'featureFlags.launchdarkly.sdkKey');
       this.setLocalSecret('idam-user-dashboard-client-secret', 'services.idam.clientSecret');
+      this.setLocalSecret('idam-user-dashboard-systemUser-password', 'services.idam.systemUser.password');
```

Comment:
Do we have the system user email in KeyVault as well? Shall we move it here too?

Reply:
yeah can do